addons/web_editor/views/snippets.xml

```diff
@@ -42,8 +42,8 @@
             <i class="fa fa-warning" /> External Image
         </we-title>
     </div>
-    <we-colorpicker t-att-string="indent and ('⌙ %s' % filter_label) or filter_label" data-set-filter="" data-no-preview="true" data-excluded="common, theme"/>
-    <we-select t-att-string="indent and ('⌙ %s' % width_label) or width_label" data-name="width_select_opt" data-no-preview="true"/>
+    <we-colorpicker t-att-string="indent and ('⌙ %s' % filter_label) or filter_label" data-set-filter="" data-excluded="common, theme"/>
+    <we-select t-att-string="indent and ('⌙ %s' % width_label) or width_label" data-name="width_select_opt"/>
```

Comment:
I thought this would be more complicated to do than just removing the no-preview flag but it seems actually ok :+1: @sdegueldre Could you confirm?

Reply:
I had initially disabled the preview on the color filter because there was the full palette of colors and naturally you would drag your mouse over all solid colors before reaching the semi-transparent ones, and for large images it would take a second or two to go through all the color previews because every preview needs to re-encode the image in JPEG. Same with width. There is no technical obstacle to enabling the preview on these options.